replica/rest/model.go

```diff
@@ -173,8 +173,6 @@ func NewReplica(context *api.ApiContext, state replica.State, info replica.Info,
 		actions["updatecloneinfo"] = true
 	case replica.Rebuilding:
 		actions["start"] = true
-		actions["resize"] = true
-		actions["snapshot"] = true
```

Comment:
let's remove resize and snapshot from other invalid states also, or put a TODO about the same